x/docs/md/intro/getting-started.md

```diff
@@ -3,7 +3,7 @@
 1. Download and configure the [command-line client](/cli).
 1. Fetch the first exercise (`exercism fetch TRACK_ID`).
 1. Write a solution to make the test suite pass.
-1. Submit your solution (`exercism submit path/to/file`).
+1. Submit your solution (`exercism submit path/to/file` or `exercism s path/to/file{.h,.cpp}`). 
```

Comment:
I asked, because different shells handle matching and globbing differently.  If you mention the example is for bash, then someone using another shell will know that they can't rely on that to work, but will have a reference.